lib/mixed-content.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Title : Prevent non https content inclusion
+ *
+ * Implementation notes : We check that every <img href=""> is using https://
+ */
+
+const ERROR = require('./doctests.js').ERROR;
+
+// The string must start by https://
+const HTTPS_URL = /^https:\/\//
+
+exports.mixedContent = {
+  name: "mixed_content_name",
+  desc: "mixed_content_desc",
+  check: function mixedContent(rootElement) {
```

Comment:
Following the other rules, this should be

`check: function checkMixedContent(rootElement) {`